modules/createRenderer.js

```diff
@@ -61,7 +67,10 @@ export default function createRenderer(config = { }) {
 
       // uses the reference ID and the props to generate an unique className
       const ruleId = renderer.ids.indexOf(rule)
-      const className = 'c' + ruleId + generatePropsReference(props)
+
+      const classNamePrefix = this && this.prettySelectors && rule.name ? rule.name + '_' : 'c'
```

Comment:
Try not to use `this` explicitly, but rather use `renderer.prettySelectors && rule.name` :P will be fine